### PR TITLE
Added Static Web Page implementation

### PR DESCRIPTION
**Pull request by primakov127**
https://primakov127.github.io/

### CODE REVIEW

File: HTML_CSS/src/css/components/banner/banner.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    text-align: center;
+    letter-spacing: 0.1em;
+
+    color: white;
+
+    font-family: "PT Sans", sans-serif;
+    font-size: 40px;
+}
+
+.banner__title span {
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/banner/banner.css]: Please do not use html tag as a selector

File: HTML_CSS/src/css/components/featured-section/featured-section.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.featured-section {
+
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/featured-section/featured-section.css]: remove empty code block

File: HTML_CSS/src/css/components/mailing/mailing.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+.mailing__title {
+    display: flex;
+    flex: 1;
+
+    margin-top: -10px;
+
+    align-items: center;
+}
+
+.mailing__title i {
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/mailing/mailing.css]: do not use html tag as a selector

File: HTML_CSS/src/css/components/mailing/mailing.css [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+.mailing__button {
+    padding: 0 20px;
+
+    cursor: pointer;
+
+    color: white;
+    border: none;
+    border-radius: 5px;
+    background-color: rgb(89, 114, 117);
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/mailing/mailing.css:74]: does the combination of border none and border-radius make sense?

File: HTML_CSS/src/css/components/portfolio/potrfolio.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    display: flex;
+    flex-direction: column;
+
+    padding-bottom: 60px;
+}
+
+.portfolio__image-container {
+    margin: 45px 0;
+    margin-bottom: 40px;
+}
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/portfolio/potrfolio.css:10]: please use short form for margin

File: HTML_CSS/src/css/components/footer/footer.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+    background-color: rgb(99, 127, 131);
+}
+
+.footer__social-container {
+    max-width: 1170px;
+    margin: 30px 0;
+    margin-right: auto;
+    margin-left: auto;
+}
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/footer/footer.css:21]: please use short form for margin

File: HTML_CSS/src/css/components/mailing/mailing.css [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+    margin-right: 30px;
+
+    justify-content: space-between;
+}
+
+.mailing__title {
+    display: flex;
+    flex: 1;
+
+    margin-top: -10px;
```
Comment (sergeygontsarevich) [inline HTML_CSS/src/css/components/mailing/mailing.css]: Please could you explain why you use a negative margin here?

**Review by sergeygontsarevich — CHANGES_REQUESTED** [review]

File: HTML_CSS/src/css/components/mailing/mailing.css (hunk 4 above)
Comment (primakov127) [reply HTML_CSS/src/css/components/mailing/mailing.css:74]: Yes, It does
It removes a browser style and adds a border-radius
The reset.css does not removes all styles properly, I guess

File: HTML_CSS/src/css/components/mailing/mailing.css (hunk 7 above)
Comment (primakov127) [reply HTML_CSS/src/css/components/mailing/mailing.css]: Yes, I could
But I've preferred to remove it and change the code
Thank you

**Review by sergeygontsarevich — APPROVED** [review]

**Pull request merged**